src/EnergyPlus/DirectAirManager.cc

```diff
@@ -472,12 +493,43 @@ namespace DirectAirManager {
 
 		// Set the ZoneNode number
 		ZoneNode = DirectAir( DirectAirNum ).ZoneSupplyAirNode;
+
+		//TRANE BEG: DCV
+		Real64 mDotFromOARequirement( 0.0 );
+		if ( ! DirectAir( DirectAirNum ).NoOAFlowInputFromUser ) {
+			mDotFromOARequirement = DirectAir( DirectAirNum ).AirMassFlowRateMax;
+			int airLoopNum( 0 );
+			Real64 airLoopOAFrac( 0.0 );
+			if ( DirectAir( DirectAirNum ).ZoneEqNum > 0) {
+				airLoopNum =ZoneEquipConfig( DirectAir( DirectAirNum ).ZoneEqNum ).AirLoopNum;
+			}
+			if ( airLoopNum > 0 ) {
+				airLoopOAFrac = DataAirLoop::AirLoopFlow( airLoopNum ).OAFrac;
+				if ( airLoopOAFrac > 0.0 ) {
+					Real64 vDotOAReq = DataZoneEquipment::CalcDesignSpecificationOutdoorAir( DirectAir( DirectAirNum ).OARequirementsPtr, DirectAir( DirectAirNum ).ZoneNum, true, true );
+					Real64 rhoAir = PsyRhoAirFnPbTdbW( Node( DirectAir( DirectAirNum ).ZoneSupplyAirNode ).Press, Node( DirectAir( DirectAirNum ).ZoneSupplyAirNode ).Temp, Node( DirectAir( DirectAirNum ).ZoneSupplyAirNode ).HumRat );
+					mDotFromOARequirement = vDotOAReq * rhoAir / airLoopOAFrac;
+					mDotFromOARequirement = min( mDotFromOARequirement , DirectAir( DirectAirNum ).AirMassFlowRateMax );
+				} else {
+					mDotFromOARequirement = DirectAir( DirectAirNum ).AirMassFlowRateMax;
+				}
```

Comment:
@mjwitte It may happen occasionally. Line 514 may not be necessary, Line 500 is:

			mDotFromOARequirement = DirectAir( DirectAirNum ).AirMassFlowRateMax;
However, I am not sure if the value should be Max or Min. If we assume it is DOAS, then it should be min.